docs/storm-kafka-client.md

```diff
@@ -345,3 +345,27 @@ Currently the Kafka spout has has the following default values, which have shown
 * offset.commit.period.ms = 30000   (30s)
 * max.uncommitted.offsets = 10000000
 <br/>
+
+# Kafka AutoCommitMode 
+
+If reliability isn't important to you -- that is, you don't care about losing tuples in failure situations --, and want to remove the overhead of tuple tracking, then you can run a KafkaSpout with AutoCommitMode.
+
+To enable it, you need to:
+* set Config.TOPOLOGY_ACKERS to 0;
+* enable *AutoCommitMode* in Kafka consumer configuration; 
+
+Here's one example to set AutoCommitMode in KafkaSpout:
+```java
+Map<String, Object> props = new HashMap<String, Object>();
+props.put(ConsumerConfig.ENABLE_AUTO_COMMIT_CONFIG, "true");
+KafkaSpoutConfig<String, String> kafkaConf = KafkaSpoutConfig
+		.builder(String bootstrapServers, String ... topics)
+		.setProp(props)
```

Comment:
nit: set prop works with a key and a value, so it would be smaller to just do
```
KafkaSpoutConfig<String, String> kafkaConf = KafkaSpoutConfig
		.builder(String bootstrapServers, String ... topics)
		.setProp(ConsumerConfig.ENABLE_AUTO_COMMIT_CONFIG, "true")
                .setFirstPollOffsetStrategy(FirstPollOffsetStrategy.EARLIEST)
		.build();
```